app/lib/methods/getRoles.ts

```diff
@@ -6,16 +6,18 @@ import log from '../../utils/log';
 import { store as reduxStore } from '../auxStore';
 import { removeRoles, setRoles as setRolesAction, updateRoles } from '../../actions/roles';
 import protectedFunction from './helpers/protectedFunction';
+import { TRoleModel } from '../../definitions';
+import sdk from '../rocketchat/services/sdk';
 
-export async function setRoles() {
+export async function setRoles(): Promise<void> {
 	const db = database.active;
 	const rolesCollection = db.get('roles');
 	const allRoles = await rolesCollection.query().fetch();
 	const parsed = allRoles.reduce((acc, item) => ({ ...acc, [item.id]: item.description || item.id }), {});
 	reduxStore.dispatch(setRolesAction(parsed));
 }
 
-export async function onRolesChanged(ddpMessage) {
+export async function onRolesChanged(ddpMessage: any) {
```

Comment:
Why any?
From what I've seen there is at least three places using this `ddpMessage` property.
Typing it would help in the future migration of this property.
You don't need to type everything, but at least passing it in this function would be nice.